front/app/containers/Admin/pages-menu/containers/HeroBanner/index.tsx

```diff
@@ -457,7 +461,10 @@ const HeroBannerForm = ({ intl: { formatMessage } }: InjectedIntlProps) => {
             onChange={handleSignedInHeaderOnChange}
           />
         </SectionField>
-        <SectionField key="avatars">
+        <SectionField
+          key="avatars"
+          data-testid="e2e-banner-avatar-toggle-section"
```

Comment:
For e2e tests, it's standard to use `data-cy` although the library doesn't strictly require it (unlike rtl). But it creates some confusion when e2e prefix is used with `data-testid` which is associated with unit tests normally. Perhaps we can change it?